utils/cache.js

```diff
@@ -4,35 +4,85 @@ Status of cache will update every time data is successfully scraped
 */
 const logger = require('./logger');
 
-exports.currentStatus = {
+var currentStatus = {
```

Comment:
What is the purpose of this variable anymore? We never use it, as we are not caching anymore.